app/script/view_model/content/InputBarViewModel.js

```diff
@@ -129,7 +113,8 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
 
     this.richTextInput = ko.pureComputed(() => {
       const mentionAttributes = ' class="input-mention" data-uie-name="item-input-mention"';
-      const pieces = this.currentMentions
+      const {text, mentions} = this.draftMessage();
```

Comment:
Because this now depends on a slightly delayed `draftMessage` observable, we will compute the new rich text based on a consistent state

Reply:
Removes timing issues with setting the `input` or the `currentMentions` one before another